### PR TITLE
Get rid of Spring dependency

### PR DESCRIPTION
**Pull request by vso-tc**

### CODE REVIEW

File: amqp/pom.xml [hunk 1]
```diff
@@ -1,42 +1,43 @@
 <?xml version="1.0"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```
Comment (vso-tc) [inline amqp/pom.xml]: Reformatted this file according to our format

**Comment by vso-tc** [comment]
Pom files were reformatted. That is why the diff is so big

File: core/pom.xml [hunk 2]
```diff
@@ -30,10 +31,6 @@
             <artifactId>commons-io</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.springframework.amqp</groupId>
```
Comment (vso-tc) [inline core/pom.xml:33]: This is the main change

**Comment by rkatsyuryna** [comment]
well done

**Pull request merged**